recipes/qt-opcua/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+if test "$CONDA_BUILD_CROSS_COMPILATION" = "1"
+then
+  CMAKE_ARGS="${CMAKE_ARGS} -DQT_HOST_PATH=${BUILD_PREFIX}"
```

Comment:
QT_HOST_PATH=${PREFIX} can be passed on native builds also (then pass CMAKE_ARGS last)
see qt-charts feedstock for example: https://github.com/conda-forge/qt-charts-feedstock/blob/main/recipe/build.sh

Reply:
Actually I used qt-charts as a reference, so I was a bit confused of why that could happen, but I see that qt-charts changed recently (after I starting working on this), I will align this recipe after those commits.